src/node_file.cc

```diff
@@ -3431,6 +3432,27 @@ static void CpSyncCopyDir(const FunctionCallbackInfo<Value>& args) {
   bool verbatim_symlinks = args[5]->IsTrue();
   bool preserve_timestamps = args[6]->IsTrue();
 
+  std::function<bool(std::string, std::string)> filter_fn = nullptr;
```

Comment:
Can you do std::optional

Reply:
done, thanks :smile: 

2ea5167158c227d4b826f9a05686e160d0c95407